src/Castle.Facilities.Synchronize/Castle.Facilities.Synchronize-VS2017.csproj

```diff
@@ -33,4 +33,8 @@
   <ItemGroup>
     <Compile Include="..\..\buildscripts\CommonAssemblyInfo.cs" />
   </ItemGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)'=='net45'">
+    <DefineConstants>$(DefineConstants);FEATURE_WINFORMS</DefineConstants>
```

Comment:
It's only needed in project Castle.Facilities.Synchronize project and in tests. I'll move it to common.props if it's preferable that way.